### ISSUE TITLE
Not syncing label colors if label exists

### ISSUE THREAD

**Issue by sc0ttkclark** [dwyl/labels#86]
If the label exists on the source repo, the color is not synced from the target repo (if the color is different).

**Comment by nelsonic** [comment]
@sc0ttkclark indeed, synching colors for _existing_ labels is a `new` requirement. 
(_PR gladly accepted_) 😉

**Comment by SimonLab** [comment]
This will be fixed with #109
Thanks for opening this issue :+1:

**Comment by SimonLab** [comment]
Fixed using https://labels.fly.dev/
